.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     machine: true
     steps:
       - checkout
-      - run: docker-compose up shellcheck
+      - run: docker-compose up --exit-code-from shellcheck shellcheck
```

Comment:
Huh... What is the default exit code if you don't specify this?

Reply:
It's non-zero only if `docker-compose` has an error internally. Otherwise, it returns `0` regardless of the containers.